packages/components/src/VirtualizedList/HeaderIcon/HeaderIcon.component.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+import { SortIndicator } from 'react-virtualized';
+
+import Icon from '../../Icon';
+import theme from './HeaderIcon.scss';
+
+/**
+ * This renderer render the header with an icon
+ * @param {object} props column header props
+ */
+function HeaderIcon({ label, sortBy, dataKey, sortDirection, columnData }) {
+	return (
+		<div className={classnames(theme['tc-header-icon'], 'tc-header-icon')}>
+			{columnData.iconName && <Icon name={columnData.iconName} title={label} />}
+			{sortBy === dataKey && <SortIndicator key="SortIndicator" sortDirection={sortDirection} />}
```

Comment:
why do you need a key here ?

Reply:
I mirrored https://github.com/bvaughn/react-virtualized/blob/master/source/Table/defaultHeaderRenderer.js
As it return an array in the default headerRenderer, the key is required. As i don't return an array, you're right, i will remove it :)